### DISCUSSION TITLE
2022년 회고 - peterkimzz

### DISCUSSION THREAD

**Discussion by giscus** [peterkimzz/blog#23] [Blog Comments]
# 2022년 회고 - peterkimzz

또 다시 돌아온 회고록이다. 아, 올해는 마스크 좀 벗길 기대했는데 언제쯤 실내 마스크를 해제할지 모르겠다.
겨울 야외 온수 풀
올초 겨울에 그 동안 정말 해보고 싶었던 것을 해봤다. 겨울에 야외 온수 풀에 가보는 것이었다. 얼굴은 춥고 몸은 따뜻한 일본 겨울 온천 같

https://www.peterkimzz.com/memoirs-by-2022/

**Comment by spilist** [comment]
재밌게 읽었습니다. 몇번 비슷하게 얘기한 것 같지만 피터를 회사로 데려온 건 2022년 최고의 선택 중 하나라고 생각해요. ㅎㅎ

**Reply by peterkimzz** [reply]
덕분에 즐거운 2022년이었습니다! 감사합니다. ㅎㅎ
